File: Dev/C++/Mascaret/src/BEHAVE/CyclicBehaviorExecution.cpp
#include "BEHAVE/CyclicBehaviorExecution.h"

namespace BEHAVE
{

CyclicBehaviorExecution::CyclicBehaviorExecution(shared_ptr<Behavior> specif,shared_ptr<InstanceSpecification> host, const Parameters& p) : SimpleBehaviorExecution(specif,host,p)
{}

CyclicBehaviorExecution::~CyclicBehaviorExecution()
{}

bool CyclicBehaviorExecution::done()
{
	return false;
}

}

File: Dev/CS/Mascaret/Mascaret/BEHAVE/CyclicBehaviorExecution.cs
using System;
using System.Collections.Generic;

namespace Mascaret
{
    public class CyclicBehaviorExecution : SimpleBehaviorExecution
    {
        public CyclicBehaviorExecution(Behavior specif, InstanceSpecification host, Dictionary<String, ValueSpecification> p)
            : base(specif, host, p)
        {
        }

        public CyclicBehaviorExecution()
        {
        }

        public override bool done()
        {
            return false;
        }
    }
}

